capabilities_matrix/_topics/infrastructure_providers.md

```diff
@@ -2,7 +2,7 @@
 
  The following tables outline the capabilities of {{ site.data.product.title_short }} that are known to work for various virtual infrastructure providers.
 
-| Discovery                                                    | vSphere | oVirt / RHV | OpenStack undercloud    | IBM Power HMC | KubeVirt |
+| Discovery                                                    | vSphere | oVirt / RHV | OpenStack undercloud    | IBM Power HMC | OpenShift Virtualization (KubeVirt) |
```

Comment:
Actually, now that I made that other comment, it should be `KubeVirt / OSV` in these docs to match the pattern of `oVirt / RHV`  (the pattern being "opensource name" / "Red Hat productized name")